node/prepublish.js

```diff
@@ -43,15 +43,14 @@ var repoUtils = require('./repoUtils');
 // Use npm.md as the README for the package.
 var absGitRepoPath = path.resolve(path.join(__dirname, '..', '..', '..'));
 var readmePath = path.join(absGitRepoPath, 'README.md');
-var readmeBackupPath = readmePath + '.orig';
+var readmeBackupPath = 'ORIGINAL.md';
```

Comment:
I could break them up to retain the path, but as long as the paths match in ```prepublish``` and ```postpublish``` we're okay, right?